test/Microsoft.Framework.ConfigurationModel.Test.Common/IConfigurationSourceExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Framework.ConfigurationModel
+{
+    public static class IConfigurationSourceExtensions
```

Comment:
It still seems somewhat dubious to me to have this extension method just for our unit tests, no? Is there a reason the main API is too difficult to use? Should the `Data` property go back to being public?